### PR TITLE
Yaml pipeline readme.

### PR DESCRIPTION
**Pull request by robertwb**
Add a basic readme to the yaml directory.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: sdks/python/apache_beam/yaml/README.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+While Beam provides powerful APIs for authoring sophisticated data
+processing pipelines, it often still has too high a barrier for
+getting started and authoring simple pipelines. Even setting up the
+environment, installing the dependencies, and setting up the project
+can be an overwhelming amount of boilerplate for some (though
+https://beam.apache.org/blog/beam-starter-projects/ has gone a long
+way in making this easier).
+
+Here we provide a simple declarative syntax for describing pipelines
+that does not require coding experience or learnig how to use an
```
Comment (twang126) [inline sdks/python/apache_beam/yaml/README.md:31]: nit typo: learnig -> learning

**Review by twang126 — APPROVED** [review]
super cool! Some very sleek examples

**Comment by robertwb** [comment]
Thanks for the quick review!

**Comment by robertwb** [comment]
retest this please

**Pull request merged**